wire-ios-request-strategy/Sources/E2EIdentity/EnrollE2EICertificateUseCase.swift

```diff
@@ -90,7 +91,9 @@ public final class EnrollE2EICertificateUseCase: EnrollE2EICertificateUseCasePro
         do {
             try await e2eiRepository.fetchTrustAnchor()
         } catch {
+            // if you fail here, why go further
             logger.warn("failed to register trust anchor: \(error.localizedDescription)")
+            throw error
```

Comment:
@KaterinaWire what do you think? should go forward if this fails here or throw?

Reply:
The initial agreement was to not throw an error and to continue enrollment. Android doesn't throw an error either.

Reply:
ok I saw this fixes too https://github.com/wireapp/wire-ios/pull/1208